lib/WebService/Stripe.pm

```diff
@@ -76,6 +86,12 @@ method refund_charge(Str $id, HashRef :$data, :$headers) {
     return $self->post( "/v1/charges/$id/refunds", $data, headers => $headers );
 }
 
+method add_source(Str $customer_id, HashRef $data, :$headers) {
```

Comment:
I named this `add_source` (rather than `create_source`) because it's not necessarily creating a new funding source - you can also pass an existing token.